src/sas/sascalc/data_util/nxsunit.py

```diff
@@ -133,24 +133,21 @@ def _build_all_units():
     # Note: temperature needs an offset as well as a scale
     temperature = _build_metric_units('kelvin','K')
     temperature.update(_build_metric_units('Kelvin','K'))
-    temperature.update(_build_metric_units('Celcius', 'C'))
-    temperature.update(_build_metric_units('celcius', 'C'))
+    temperature.update(_build_metric_units('Celcius', '℃'))
+    temperature.update(_build_metric_units('celcius', '℃'))
```

Comment:
build_metric_units adds the milli/centi/kilo/... prefixes.  Does m℃ make sense as a unit? See code in #1575 for alternate handling of degree units.

Don't need 'Kelvin' or 'Hertz' since you are normalizing case in standardize_units.

Conversion between celsius and kelvin is incorrect.  I added ticket #1575 for this.

While you are in the file, add `μ=1e-6` to the short prefixes.